config/routes.rb

```diff
@@ -279,6 +279,7 @@
 
   # You can have the root of your site routed with "root
   get "/about" => "pages#about"
+  get "/crayons" => "pages#crayons"
```

Comment:
TOP!! We may want to double check if it's fine for this page to be public. +1 for me

Reply:
Good point. What would you suggest instead?

Reply:
to ask @lisasy and PBJ if it's fine if this page stays public :)

Reply:
I've changed it to not be top-level, it's now `/page/crayons`